libs/order/src/selectors/order-entities.selector.ts

```diff
@@ -124,80 +194,84 @@ const createOrderEntitySelectors = <T extends DaffOrder = DaffOrder>() => {
     }
   );
 
+  const selectOrderGrandTotal = createSelector(
+    selectEntities,
+    (orders, props) => {
+      const totals = selectOrderTotals.projector(orders, {id: props.id});
+      const index = totals.findIndex(total => total.type === DaffOrderTotalTypeEnum.GrandTotal);
+
+      return index > -1 ? totals[index] : null
+    }
+  );
+  const selectOrderSubtotal = createSelector(
+    selectEntities,
+    (orders, props) => {
+      const totals = selectOrderTotals.projector(orders, { id: props.id });
+      const index = totals.findIndex(total => total.type === DaffOrderTotalTypeEnum.Subtotal);
+
+      return index > -1 ? totals[index] : null
+    }
+  );
+  const selectOrderShipping = createSelector(
+    selectEntities,
+    (orders, props) => {
+      const totals = selectOrderTotals.projector(orders, { id: props.id });
+      const index = totals.findIndex(total => total.type === DaffOrderTotalTypeEnum.Shipping);
+
+      return index > -1 ? totals[index] : null
+    }
+  );
+  const selectOrderDiscount = createSelector(
+    selectEntities,
+    (orders, props) => {
+      const totals = selectOrderTotals.projector(orders, { id: props.id });
+      const index = totals.findIndex(total => total.type === DaffOrderTotalTypeEnum.Discount);
+
+      return index > -1 ? totals[index] : null
```

Comment:
what about:
```
return selectOrderTotals.projector(orders, { id: props.id })
   .find(total => total.type === DaffOrderTotalTypeEnum.Discount);
```

Reply:
I don't think we can use `Array#find` because of IE11 compatibility, right?

Reply:
hmm...that does sound like a thing. Though we do use it elsewhere, maybe we shouldn't be.

Reply:
yeah I often forget that we can't use new features. We honestly should just have a polyfill for IE